packages/sdk/dappkit/README.md

```diff
@@ -136,21 +136,24 @@ import {
   waitForSignedTxs
 } from "@celo/dappkit";
 
-// Let's assume that the address has enough cUSD to pay the
-// transaction fees of all the transactions and enough to buy 10 CELO
-// AND it's already a registered Account (otherwise it will require a call
-// to the `createAccount` method from the Accounts contract)
+const dappName = "My DappName";
+const callback = Linking.makeUrl("/my/path");
+
+// Let's assume that the address has funds enough in cUSD to pay the
+// transactions fees of all the transactions and enough to buy 10 CELOs
+// AND it's already a registered Account (otherwise it will require to call
+// the `createAccount` method from the Accounts contract)
 
 // We will be using the following contracts:
 const stableToken = await kit.contracts.getStableToken();
 const exchange = await kit.contracts.getExchange();
 const lockedGold = await kit.contracts.getLockedGold();
-const election = await kit.contracts.getElection()
+const election = await kit.contracts.getElection();
 
 const decimals = await stableToken.decimals(); // both cusd and celo use the same
 
 const tenCelo = new BigNumber(10).pow(parseInt(decimals, 10)).toString();
```

Comment:
I think `decimals` already returns a number, could we simplify these lines to: 
```
new BigNumber(10).pow(decimals).toString()
new BigNumber(100).pow(decimals).toString()
```

Reply:
Changed it to one and ten (to be able to do everything with one faucet) and those are: `BigNumber("1e18")` and `BigNumber("10e18")`
`BigNumber(100).pow(decimals)` this is 100^18, haha, and what we should do in this cases is `BigNumber(10).multipliedBy(BigNumber(10).pow(18))`